solr/core/src/java/org/apache/solr/core/SolrXmlConfig.java

```diff
@@ -568,4 +568,17 @@ private static PluginInfo getTracerPluginInfo(XmlConfigFile config) {
     Node node = config.getNode("solr/tracerConfig", false);
     return (node == null) ? null : new PluginInfo(node, "tracerConfig", false, true);
   }
+  
+  private static SharedStoreConfig loadSharedStoreConfig(NamedList<Object> nl) {
+    if (nl == null) {
+      // shared store is not configured
+      return null;
+    } 
+    boolean enabled = Boolean.parseBoolean(
+        required("sharedStore", "sharedStoreEnabled", removeValue(nl, "sharedStoreEnabled")));
+    if (enabled) {
+      return new SharedStoreConfig();
+    }
```

Comment:
Maybe it works for now, but later we might want to properly add an enabled property to SharedStoreConfig and create SharedStoreConfig even enabled is false.